platform/security/filter/security-filter-delegate/src/main/java/org/codice/ddf/security/filter/delegate/DelegateServletFilter.java

```diff
@@ -1,123 +0,0 @@
-/**
- * Copyright (c) Codice Foundation
- * <p>
- * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
- * General Public License as published by the Free Software Foundation, either version 3 of the
- * License, or any later version.
- * <p>
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
- * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
- * is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- */
-package org.codice.ddf.security.filter.delegate;
-
-import java.io.IOException;
-import java.util.Iterator;
-import java.util.LinkedList;
-
-import javax.servlet.Filter;
-import javax.servlet.FilterChain;
-import javax.servlet.FilterConfig;
-import javax.servlet.ServletException;
-import javax.servlet.ServletRequest;
-import javax.servlet.ServletResponse;
-import javax.servlet.http.HttpServletRequest;
-
-import org.apache.commons.lang.StringUtils;
-import org.codice.ddf.security.policy.context.ContextPolicyManager;
-import org.osgi.framework.BundleContext;
-import org.osgi.framework.InvalidSyntaxException;
-import org.osgi.framework.ServiceReference;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- * {@link DelegateServletFilter} is meant to detect any Security ServletFilters
- * and call their {@link FilterChain}. If none are found it becomes a pass
- * through filter.
- */
-public class DelegateServletFilter implements Filter {
```

Comment:
Why did you remove the `DelegatingServletFilter`?  This is part of a solution to get in front of servlet filters defined in web.xml.  For example, without this it is not possible to properly secure the Solr servlet filters.

👎  until this is resolved.